openrndr-core/src/main/kotlin/org/openrndr/shape/Segment.kt

```diff
@@ -6,26 +6,26 @@ import org.openrndr.shape.internal.BezierQuadraticSampler2D
 import kotlin.math.*
 
 /**
- * Segment describes a linear or bezier path between two points
+ * Creates a new [Segment], which specifies a linear or a Bézier curve path between two control points.
```

Comment:
This is a matter requiring a bit of discussion, because from a mathematical sense, they're all control points, are they not? Yet it's definitely common to refer to the intermediate control points as just "control points" and the outer control points as "end points". I'll think about it, might be worth talking about it on Slack?